src/bot/commands/infraction.ts

```diff
@@ -338,11 +338,21 @@ async function initExistingInfractionData(server: Guild) {
 function setupMutedChannelSettings(server: Guild, mutedRole: Role) {
     server.channels.cache.forEach(channel => {
         if (channel.type === "text") {
-            (channel as TextChannel).createOverwrite(mutedRole, { "SEND_MESSAGES": false, "ADD_REACTIONS": false });
-        }
-
-        if (channel.type == "voice") {
-            (channel as VoiceChannel).createOverwrite(mutedRole, { "SPEAK": false, "STREAM": false });
+            const mutedTextPermissions = channel.permissionOverwrites.get(mutedRole.id);
+            console.log(mutedTextPermissions?.toJSON());
```

Comment:
Let's remove this debug console.log before merging

Reply:
Yep, removed.